packages/client/src/components/TransactionList.js

```diff
@@ -1,78 +1,47 @@
 import React from "react";
-import { useModalContext } from "../contexts";
-import { getFlowscanUrlForTransaction, shortenAddr } from "../utils";
+import { useModalContext } from "contexts";
 import TransactionStatusIcon from "./TransactionStatusIcon";
 
-const FALLBACK_RECIPIENT_NAME = "No name";
+const maxWidths = {
+  index: 50,
+  status: 140,
+  date: 160,
+  action: 70,
+};
 
-function getDisplayStringForTokenTransfer(tokenTransfer) {
-  if (!tokenTransfer) {
-    return "";
-  }
+const Column = ({ className = "", children, style = {} }) => (
```

Comment:
Is this component purely for styling? What's the consideration of rendering by columns not rows?

Reply:
was to reduce the boilerplate of `column is-flex is-align-items-center`. I guess we could rename `TransactionListItem` to `TransactionListRow` or simply `Row`

Reply:
Yeah and looks like only 2 out of 5 columns need `is-flex is-align-items-center`